sdk/cosmos/tests/user.rs

```diff
@@ -39,15 +39,16 @@ async fn users() {
         .await
         .unwrap();
 
+    let list_users_response = database_client.list_users().execute().await.unwrap();
+    assert_eq!(list_users_response.users.len(), 0);
```

Comment:
This fails because the expected output is 1, not 0 (replace user must keep the same number of users upon success). 

```suggestion
    assert_eq!(list_users_response.users.len(), 1);
```